api/src/services/authentication.ts

```diff
@@ -358,7 +358,8 @@ export class AuthenticationService {
 		}
 
 		const newRefreshToken = nanoid(64);
-		const refreshTokenExpiration = new Date(Date.now() + getMilliseconds(env['REFRESH_TOKEN_TTL'], 0));
+		const sessionDuration = env[options?.session ? 'SESSION_COOKIE_TTL' : 'REFRESH_TOKEN_TTL'];
+		const refreshTokenExpiration = new Date(Date.now() + getMilliseconds(sessionDuration, 0));
```

Comment:
Should it be `tokenDuration` here since we're not always in the session context?

Reply:
True!

Reply:
Whether it's applicable to a `session token` or not this still applies to the "duration of a directus_session in the database". the duration is not actually related to a token in both situations but is a session in both.

Reply:
I kinda wanted to just use `TTL` but we're already using that elsewhere in the function 😂